hybridesmbench/eval/_diags/timeseries/data/asr/metadata.yml

```diff
@@ -1,13 +1,42 @@
-? asr/CMIP6_ACCESS-CM2_Amon_historical_r1i1p1f1_asr_gn_19790101-20141231.nc
+? /scratch/b/b309059/esmvaltool_output/recipe_timeseries_AMIP_20251027_111027/preproc/timeseries/asr/CMIP6_ACCESS-CM2_Amon_amip_r1i1p1f1_asr_gn_19950101-20141231.nc
 : activity: CMIP
-  alias: CMIP6_ACCESS-CM2
+  alias: CMIP6_ACCESS-CM2_amip
+  dataset: ACCESS-CM2
+  derive: true
+  diagnostic: timeseries
+  end_year: 2014
+  ensemble: r1i1p1f1
+  exp: amip
+  filename: /scratch/b/b309059/esmvaltool_output/recipe_timeseries_AMIP_20251027_111027/preproc/timeseries/asr/CMIP6_ACCESS-CM2_Amon_amip_r1i1p1f1_asr_gn_19950101-20141231.nc
+  force_derivation: true
+  frequency: mon
+  grid: gn
+  institute: CSIRO-ARCCSS
+  long_name: Absorbed shortwave radiation
+  mip: Amon
+  modeling_realm:
+  - atmos
+  preprocessor: create_global_annual_mean_time_series
+  project: CMIP6
+  recipe_dataset_index: 70
+  short_name: asr
+  standard_name: ''
+  start_year: 1995
+  timerange: 19950101/20141231
+  title: Global Mean Absorbed Shortwave Radiation
+  units: W m-2
+  variable_group: asr
+  version: v20191108
+? /scratch/b/b309059/esmvaltool_output/recipe_timeseries_AMIP_20251027_111027/preproc/timeseries/asr/CMIP6_ACCESS-CM2_Amon_historical_r1i1p1f1_asr_gn_19790101-20141231.nc
+: activity: CMIP
+  alias: CMIP6_ACCESS-CM2_historical
   dataset: ACCESS-CM2
   derive: true
   diagnostic: timeseries
   end_year: 2014
   ensemble: r1i1p1f1
   exp: historical
-  filename: asr/CMIP6_ACCESS-CM2_Amon_historical_r1i1p1f1_asr_gn_19790101-20141231.nc
+  filename: /scratch/b/b309059/esmvaltool_output/recipe_timeseries_AMIP_20251027_111027/preproc/timeseries/asr/CMIP6_ACCESS-CM2_Amon_historical_r1i1p1f1_asr_gn_19790101-20141231.nc
```

Comment:
Hardcoded absolute paths instead of relative paths, and some are still called historical instead of amip?